pkgs/development/tools/misc/debugedit/default.nix

```diff
@@ -0,0 +1,30 @@
+#TODO@deliciouslytyped The tool seems to unnecessarily force mutable access for the debugedit `-l` feature
+{fetchgit, lib, stdenv, autoreconfHook, pkgconfig, elfutils, help2man, util-linux}:
```

Comment:
`pkgconfig` is only an alias, so use `pkg-config` instead.  That's why OfBorg complains.